File: Modul 5/Soal 2/PRAK502-2410817210019-AchmadReihanAlfaiz.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2)
{
     return nilai1 - nilai2;
}

int mutlak(int angka)
{
     return abs(angka);
}

int main()
{
     int a, b, c, d;

     scanf("%d", &a);
     scanf("%d", &c);
     scanf("%d", &b);
     scanf("%d", &d);

     int Hasil = mutlak(hitung(a, b)) + mutlak(hitung(c, d));
     printf("%d", Hasil);

     return 0;
}

File: Modul 5/Soal 2/PRAK502-2410817210019-AchmadReihanAlfaiz.py
def hitung(nilai1, nilai2):
    return nilai1 - nilai2


def mutlak(angka):
    return abs(angka)


a, c, b, d = map(int, input().split())

hasil = mutlak(hitung(a, b)) + mutlak(hitung(c, d))
print(hasil)
